sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleFacadeEjb.java

```diff
@@ -729,7 +727,8 @@ public void validate(SampleDto sample) throws ValidationRuntimeException {
 		Pseudonymizer pseudonymizer = Pseudonymizer.getDefault(userService::hasRight, I18nProperties.getCaption(Captions.inaccessibleValue));
 
 		for (SampleExportDto exportDto : resultList) {
-			List<PathogenTest> pathogenTests = sampleService.getById(exportDto.getId()).getPathogenTests();
+			Sample sample1 = sampleService.getById(exportDto.getId());
```

Comment:
please rename `sample1` to `sample`

Reply:
Variable sample is already defined in that scope.

Reply:
Then rename also the other one